src/docs/contributing/approach/sdk-docs/write-performance.mdx

```diff
@@ -31,15 +31,21 @@ This file is `index.mdx`. It explains how to install or set up (if appropriate)
+1. If the SDK is still a work-in-progress and is exposed as an experimental or an early access feature, add the platform to the `<PlatformSection>` wrapping the appropriate alert located at the beginning of the content for the page.
 
-2. Add a code sample to `/src/includes/performance/configure-sample-rate/` that covers how to set sample rates.
+1. If the SDK is expected to be used in a high-throughput environment, add the platform to the following `<PlatformSection>` which advises testing before deploying to production.
 
-3. Determine if you want to have users verify using our custom instrumentation. If so, add the SDK to `Supported` list in the `<PlatformSection>`. If not, add the SDK to `notSupported` list, which will prevent the `<PlatformLink>` to the Custom Instrumentation page from displaying for this SDK.
+1. If the SDK includes automatic and custom instrumentation, add your platform to the next `<PlatformSection>` that wraps around text that discusses this.
 
-4. Determine if the paragraph regarding pageload and navigation applies to the SDK. If so, add the SDK to `Supported` list in the `<PlatformSection>`. If not, add the SDK to `notSupported` list, which will prevent the content from displaying for this SDK.
+1. Determine if the SDK needs an installation step or a property set to enable tracing. If so, add the SDK to `Supported` list in the `<PlatformSection>` surrounding the **Enable Tracing** heading, then add a code sample to `performance/enable-tracing/`. If not, add the SDK to `notSupported` list, which will prevent the **Enable Tracing** heading from displaying for this SDK.
 
-5. Determine the level of instrumentation support for verification and add as appropriate to the `Supported` list in the `<PlatformSection>`.
+1. Add a code sample to `/src/includes/performance/configure-sample-rate/` that covers how to set sample rates. Make sure to update `/platforms/common/configuration/`.
+
+1. If automatic instrumentation is supported, add your platform to the `<PlatformSection>` immediately following **Verify** which advises users to either use auto or custom (a.k.a. manual) instrumentation to verify the feature. Either omit the platform from `supported` or  the platform to `notSupported` if auto instrumentation is not available.
+
+1. If automatic instrumentation is not supported, add your platform to the second `<PlatformSection>` after **Verify** which advises users to test out tracing using custom (manual) instrumentation. Omit the platform from `supported` or add it to `notSupported` if automatic instrumentation is available.
+
+1. Determine if the paragraph regarding pageload and navigation applies to the SDK. If so, add the SDK to `Supported` list in the `<PlatformSection>` surrounding **Connecting Services**. If not, add the SDK to `notSupported` list, which will prevent the content from displaying for this SDK.
```

Comment:
Thanks for catching this, did a bit of digging and it turns out this paragraph actually refers to the `PlatformSection` that I skipped. I'll need to add a dedicated point for connecting services.

Reply:
I've decided to avoid covering Connecting Services. It's likely to either to apply to all of the SDKs, or to only very specific SDKs. Sections that are very specific to certain SDKs already are unaddressed.

It's not too clear by the terminology what it means, at least to me personally. Based on the content, it sounds like it's related to something that we internally refer to as Distributed Tracing and is meant for developers using more than one SDK, where one of those SDKs is JS. The instructions seem to be explaining how to hook up the JS SDK's traces with other SDKs. I'm not too sure why this isn't exclusive to the JS documentation, plus the section hardcodes links to JS documentation. Regardless of whatever the situation might be, I think it's something I can leave out of this PR since it's turning into a bit of a rabbit hole.